src/lib/parsers/javascript/javascript.ts

```diff
@@ -0,0 +1,89 @@
+import * as eslint from 'eslint';
+import * as espree from 'espree';
+
+import { determineMediaTypeForScript } from '../../utils/content-type';
+import { IAsyncHTMLElement, IElementFound, IFetchEnd, IScriptParse, Parser } from '../../types';
+import { Sonarwhal } from '../../sonarwhal';
+
+const scriptContentRegex: RegExp = /^<script[^>]*>([\s\S]*)<\/script>$/;
+const defaultParserOptions = {
+    comment: true,
```

Comment:
We should add a comment why we are using these options by default, and maybe a link?